openquake/mbt/tools/fault_modeler/README.md

```diff
@@ -0,0 +1,170 @@
+# Fault Model Building Tools
+
+This module includes a set of functionalities to create an OpenQuake source model from an active fault database in geojson format.
+
+## Getting Started
+
+The fault model builder can be run as a Python module or directly from the command line.
+
+### In Python
+
+As a module, the main function *build_fault_model* can be called with:
+
+```python
+from openquake.mbt.tools.fault_modeler.fault_source_modeler import build_fault_model
+
+build_fault_model(geojson_file='FaultDatabase.geojson', xml_output='FaultModel.xml')
+```
+
+or, alternatively, by passing a configuration file (.ini):
+
+```python
+build_fault_model(cfg_file='config.ini')
+```
+
+Note that if the *xml_output* argument is not specified (either as an argument of in the .ini file), the code returns a list of the generated OpenQuake fault source objects as output.
+
+### From the Command Line
+
+The above examples can also be run as a shell command:
+
+```console
+fault_source_modeler.py -geo FaultDatabase.geojson -xml FaultModel.xml
+```
+or identically:
+
+```console
+fault_source_modeler.py -cfg config.ini
+```
+
+(Type ```fault_source_modeler.py -h``` to get a list of the available options)
+
+### Optional Parameters
+
+Several optional parameters can be passed, either as arguments or within the configuration file.
+Modificable parameters are:
+
+| Key | Description | Default Value |
+|:----|:------------|:--------|
+| b_value | ... | ... |
+| M_min | ... | ... |
+| M_max | ... | ... |
+| bin_width | ... | ... |
+| aseismic_coefficient | ... | ... |
+| rupture_aspect_ratio | ... | ... |
+| rupture_mesh_spacing | ... | ... |
+| minimum_fault_length | ... | ... |
+| tectonic_region_type | ... | ... |
+| upper_seismogenic_depth | ... | ... |
+| lower_seismogenic_depth | ... | ... |
+| magnitude_scaling_relationship | ... | ... |
+
+A single parameter can be passed as optional argument like:
+
+```python
+build_fault_model(geojson_file='FaultDatabase.geojson', 
+                  aseismic_coefficient=0.9,
+                  lower_seismogenic_depth=25.)
+```
+
+(note that passing single parameters as flags is presently not yet implemented in the command line interface)
```

Comment:
``flags`` => ``explicit console arguments`` flags are normally associated with ``minus``+``single-character``.

Reply:
This is an issue related to the use of the sap module. For the moment it is ok like this, we will then replace it in the future.